server/src/test/java/org/elasticsearch/index/mapper/DynamicTemplatesTests.java

```diff
@@ -685,16 +682,8 @@ public void testTemplateWithoutMatchPredicates() throws Exception {
         }
         mapping.endObject();
         MapperService mapperService = createMapperService(mapping);
-        ParsedDocument doc = mapperService.documentMapper()
-            .parse(
-                new SourceToParse(
-                    "1",
-                    new BytesArray("{\"foo\": \"41.12,-71.34\", \"bar\": \"41.12,-71.34\"}"),
-                    XContentType.JSON,
-                    null,
-                    Map.of("foo", "geo_point")
-                )
-            );
+        ParsedDocument doc = mapperService.documentMapper().parse(new SourceToParse("1", new BytesArray("""
+            {"foo": "41.12,-71.34", "bar": "41.12,-71.34"}"""), XContentType.JSON, null, Map.of("foo", "geo_point")));
```

Comment:
In my opinion, this is a case where it's clearer to extract the JSON:
```suggestion
        final String json = """
            {"foo": "41.12,-71.34", "bar": "41.12,-71.34"}
            """;
        ParsedDocument doc = mapperService.documentMapper()
            .parse(new SourceToParse("1", new BytesArray(json), XContentType.JSON, null, Map.of("foo", "geo_point")));
```